### ISSUE TITLE
selenium's Python wheel distribution contains duplicate files and may fail to install

### ISSUE THREAD

**Issue by wiggin15** [SeleniumHQ/selenium#5907]
## Meta -
OS:  
Windows
Selenium Version:  
3.12.0

## Expected Behavior -
The selenium wheel file should be installable with "setuptools".

## Actual Behavior -
Using `setuptools 39.1.0` on Windows to install the selenium Python wheel distribution (`selenium-3.12.0-py2.py3-none-any.whl`) fails with:

```
error: [Error 183] Cannot create a file when that file already exists
```

Reason: the selenium wheel file contains the following files in the wheel file:

```
selenium/webdriver/firefox/amd64/x_ignore_nofocus.so
selenium-3.12.0.data/data/selenium/webdriver/firefox/amd64/x_ignore_nofocus.so
```

setuptools does not handle this well on Windows, and the project maintainers claims it's the wheel file that's invalid (even though `pip` successfully installs the wheel, silently overriding duplicate files). See https://github.com/pypa/setuptools/pull/1371. 
Possible resolution: `data_files` in `setup.py` should not include files that are already packaged.

## Steps to reproduce -
```
> wget https://files.pythonhosted.org/packages/57/bc/17164fd471ccdf0df3a992c710c0c3c47743462ff41ab72a02c6ede96e90/selenium-3.12.0-py2.py3-none-any.whl
> easy_install selenium-3.12.0-py2.py3-none-any.whl
Processing selenium-3.12.0-py2.py3-none-any.whl
Installing selenium-3.12.0-py2.py3-none-any.whl to c:\cygwin\home\administrator\python\lib\site-packages
error: [Error 183] Cannot create a file when that file already exists
```

**Comment by lmtierney** [comment]
@davehunt thoughts?

**Comment by davehunt** [comment]
It sounds like a packaging issue, I don't see why we'd need to have duplicate files in the distribution. It won't be long before we can probably remove these files altogether, as they're not used for recent versions of Firefox.

**Comment by lmtierney** [comment]
Closed via #5982
